packages/storage-postgres/migrations/0059_signal_task_fields.sql

```
@@ -0,0 +1,16 @@
+ALTER TABLE "coherences" ADD COLUMN "due_at" timestamp with time zone;
+ALTER TABLE "coherences" ADD COLUMN "progress_status" text;
+ALTER TABLE "coherences" ADD COLUMN "board" text;
+ALTER TABLE "coherences" ADD COLUMN "assignee_ids" jsonb DEFAULT '[]'::jsonb NOT NULL;
+
+ALTER TABLE "spaces" ADD COLUMN "signal_workflow" jsonb DEFAULT '{}'::jsonb NOT NULL;
+
+ALTER TABLE "space_scheduled_items" ADD COLUMN "coherence_id" integer;
+ALTER TABLE "space_scheduled_items" ADD CONSTRAINT "space_scheduled_items_coherence_id_coherences_id_fk"
+  FOREIGN KEY ("coherence_id") REFERENCES "coherences"("id") ON DELETE set null ON UPDATE no action;
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)